### DISCUSSION TITLE
Raw file from source (e.g. .PDF/.DOCX/.PPTX/.TXT etc from GDrive/S3/sharepoint/box etc)

### DISCUSSION THREAD

**Discussion by songohannyc** [airbytehq/airbyte#34760] [Connector Questions]
Hi,

Newbie here. Is it possible to download raw bytes of PDF from GDrive and other using Airbyte API? We don't need Airbyte to extract the text, we have our own ETL to do that afterward. 

For example, Paragon support this https://docs.useparagon.com/resources/integrations/google-drive#building-google-drive-workflows (Get File by ID)

Does anyone know how Airbyte support this functionality? 

Thank you.

**Comment by sajarin** [comment]
Currently, Airbyte's Google Drive source connector does not support downloading raw bytes of files. It is designed to extract text content from files [source](https://docs.airbyte.com/integrations/sources/google-drive#set-up-the-google-drive-source-connector-in-airbyte).

[a comment by an automated account is not shown]
